pkg/retry/retry.go

```diff
@@ -81,6 +82,13 @@ func IsErrorRetryable(err error) bool {
 			return false
 		}
 		return true
+	case *docker.UnexpectedHTTPStatusError:
```

Comment:
(In the c/image PR I’m suggesting that not using a pointer type would be better.)